### PR TITLE
adding in opensearch firehose

### PR DESCRIPTION
**Pull request by JasonTheMain**
## Changes proposed in this pull request:
- addition of new secret for opensearch-firehose
- addition of logsearchv2-firehose for rotation of logsearchv1
-

## security considerations
opensearch firehose secret will replace logsearch firehose when opensearch replaces.

### CODE REVIEW

File: bosh/opsfiles/clients.yml [hunk 1]
```diff
@@ -6,7 +6,16 @@
     access-token-validity: 172800
     authorized-grant-types: client_credentials
     authorities: doppler.firehose,cloud_controller.global_auditor
-    secret: ((logsearch-firehose-ingestor-client-secret))
+    secret: ((logsearchv2-firehose-ingestor-client-secret))
```
Comment (ChrisMcGowan) [inline bosh/opsfiles/clients.yml:9]: Leave this back to the OG value
    secret: ((logsearch-firehose-ingestor-client-secret))

File: bosh/opsfiles/clients.yml [hunk 2]
```diff
@@ -6,7 +6,16 @@
...
+
```
Comment (ChrisMcGowan) [inline bosh/opsfiles/clients.yml:10]: # New legacy ES firehose client - added v2 to client name and secret path - roll legacy deployments to use this client/secret
- type: replace
  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaa/clients/logsearchv2_firehose_ingestor?
  value:
    override: true
    access-token-validity: 172800
    authorized-grant-types: client_credentials
    authorities: doppler.firehose,cloud_controller.global_auditor
    secret: ((logsearchv2-firehose-ingestor-client-secret))

File: bosh/opsfiles/clients.yml
Comment (cweibel) [inline bosh/opsfiles/clients.yml:1]: Are you manually generating the secret and uploading to credhub?  You could add the secret as a variables block like https://github.com/cloud-gov/cg-deploy-cf/commit/733f245f3d753705c59ec64eb4244148347e1231

**Review by cweibel — APPROVED** [review]
Variables are good, paths look correct, change to pipeline order of ops files should work

**Pull request merged**